media_service/iiif.py

```diff
@@ -240,7 +245,7 @@ def __init__(self, manifest, canvas_id):
         self.resource = None
 
     def add_image(self, image):
-        self.resource = IIIFImageResource(self.manifest, image['id'], image['url'], image['is_link'])
+        self.resource = IIIFImageResource(self.manifest, self.id, image['url'], image['is_link'])
```

Comment:
Yes, you're correct. Structuring it so that we only ever have one image per canvas means that Mirador will only really function as a slide viewer. It's possible that down the line we might have to change this decision and make it more flexible.